app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -2,21 +2,26 @@ module Api
   module V1x0
     module Mixins
       module RBACMixin
-
         def write_access_check
-          access_check('write')
+          resource_check('write')
         end
 
         def read_access_check
-          access_check('read')
+          resource_check('read')
         end
 
-        def access_check(verb)
+        def resource_check(verb, id = params[:id], klass = controller_name.classify.constantize)
           return unless RBAC::Access.enabled?
           access_obj = RBAC::Access.new(controller_name.classify.constantize.table_name, verb).process
-          raise Catalog::NotAuthorized, "#{verb.titleize}  access not authorized for #{controller_name.classify.constantize}" unless access_obj.accessible?
+          raise Catalog::NotAuthorized, "#{verb.titleize}  access not authorized for #{klass}" unless access_obj.accessible?
```

Comment:
Same here, guessing this is the source of the error since it was just a simple variable name change and the two spaces existed before.